changelogs/CHANGELOG.rst

```diff
@@ -5,20 +5,31 @@ Arista Eos Collection Release Notes
 .. contents:: Topics
 
 
-v1.3.0
+v2.0.0
 ======
 
-Deprecated Features
--------------------
```

Comment:
Is the changelog correctly generated? Looks like the deprecated features section from 1.3.0 is removed?